lib/postgrex/error.ex

```diff
@@ -20,4 +20,14 @@ defmodule Postgrex.Error do
   def exception(arg) do
     super(arg)
   end
+
+  def exception(:ssl, message, reason) do
+    reason = :ssl.format_error(reason)
+    %Postgrex.Error{message: "ssl #{message}: #{reason}"}
+  end
+
+  def exception(:tcp, message, reason) do
+    reason = :inet.format_error(reason)
+    %Postgrex.Error{message: "tcp #{message}: #{reason}"}
+  end
```

Comment:
This still hasn't been addressed. `ssl` will give a term not atom though in many cases.